src/vs/workbench/contrib/welcome/gettingStarted/common/gettingStartedContent.ts

```diff
@@ -395,5 +396,158 @@ export const content: GettingStartedContent = [
+						command: 'workbench.action.debug.selectandstart'
+					},
+					doneOn: { commandExecuted: 'workbench.action.debug.selectandstart' },
+					media: {
+						type: 'image', altText: 'Run and debug view.', path: {
+							dark: 'dark/debug.png',
+							light: 'light/debug.png',
+							hc: 'light/debug.png'
+						},
+					},
+				},
+				{
+					id: 'scmClone',
+					title: localize('gettingStarted.scm.title', "Track your code with Git"),
+					description: localize('gettingStarted.scmSetup.description', "Set up the built-in version control for your project to track your changes and collaborate with others."),
+					when: 'config.git.enabled && !git.missing && workspaceFolderCount == 0',
+					button: {
+						title: localize('gettingStarted.scmClone.button', "Clone Repository"),
+						command: 'git.clone'
+					},
+					doneOn: { commandExecuted: 'git.clone' },
+					media: {
+						type: 'image', altText: 'Source Control view.', path: {
+							dark: 'dark/scm.png',
+							light: 'light/scm.png',
+							hc: 'light/scm.png'
+						},
+					},
+				},
+				{
+					id: 'scmSetup',
+					title: localize('gettingStarted.scm.title', "Track your code with Git"),
+					description: localize('gettingStarted.scmSetup.description', "Set up the built-in version control for your project to track your changes and collaborate with others."),
+					when: 'config.git.enabled && !git.missing && workspaceFolderCount != 0 && gitOpenRepositoryCount == 0',
+					button: {
+						title: localize('gettingStarted.scmSetup.button', "Initialize Git Repository"),
+						command: 'git.init'
+					},
+					doneOn: { commandExecuted: 'git.init' },
+					media: {
+						type: 'image', altText: 'Source Control view.', path: {
+							dark: 'dark/scm.png',
+							light: 'light/scm.png',
+							hc: 'light/scm.png'
+						},
+					},
+				},
+				{
+					id: 'scm',
+					title: localize('gettingStarted.scm.title', "Track your code with Git"),
+					description: localize('gettingStarted.scm.description', "No more looking up Git commands! Git and GitHub workflows are seamlessly integrated."),
+					when: 'config.git.enabled && !git.missing && workspaceFolderCount != 0 && gitOpenRepositoryCount != 0 && activeViewlet != \'workbench.view.scm\'',
+					button: {
+						title: localize('gettingStarted.scm.button', "Open Source Control"),
+						command: 'workbench.view.scm'
+					},
+					doneOn: { commandExecuted: 'workbench.view.scm.focus' },
+					media: {
+						type: 'image', altText: 'Source Control view.', path: {
+							dark: 'dark/scm.png',
+							light: 'light/scm.png',
+							hc: 'light/scm.png'
+						},
+					},
+				},
+				{
+					id: 'tasks',
+					title: localize('gettingStarted.tasks.title', "Automate your project tasks"),
+					when: 'workspaceFolderCount != 0',
+					description: localize('gettingStarted.tasks.description', "Configure your workflows as tasks and enjoy the fully integrated experience of running scripts and monitoring results."),
```

Comment:
Suggestion:
"Create tasks for your common workflows and enjoy the integrated experience of running scripts and automatically checking results."